.github/workflows/workflow.yaml

```diff
@@ -43,16 +43,16 @@ jobs:
     steps:
     - uses: actions/checkout@v4
       with:
-        fetch-depth: 0 #required to be able to generate prerelease tags
+        fetch-depth: 0 # Required to be able to generate prerelease tags
     - uses: actions/setup-go@v5
       with:
         go-version-file: go.mod
-    - uses: google-github-actions/auth@v2
+    - name: Build
+      run: make debug-workflow all
```

Comment:
no strong opinion about but is `debug-workflow` intentionally here ?

Reply:
Yes, intentional while working on the eng-plan project for latest/dev release channel stuff.